sphinx_qt_documentation/utils.py

```diff
@@ -52,7 +52,8 @@ def _get_signal_and_version():
 
 Signal, QT_VERSION = _get_signal_and_version()
 SIGNAL_PREFIXES_REGEX = "".join(
-    rf"(?:{name}\.)?" for name in Signal.__module__.split(".")
+    rf"(?:{Signal.__module__.split('.', i)[-1]}\.)?"
+    for i in range(len(Signal.__module__.split(".")))
```

Comment:
I changed the regex so that it would match `PyQt5.QtCore` or `QtCore` but not `PyQt5` alone.
I split with i ranging from 0 to number of points in the module name and take the last part.
That gives us "PyQt5.QtCore" and then "QtCore", and we optionaly match one of these.